web_src/js/types.ts

```diff
@@ -59,3 +59,20 @@ export type FomanticInitFunction = {
   settings?: Record<string, any>,
   (...args: any[]): any,
 }
+
+export type CommitStatus = 'pending' | 'success' | 'error' | 'failure' | 'warning';
+
+export type CommitStatusMap = {
+  [status in CommitStatus]: {
+    name: string,
+    color: string,
+  };
+};
+
+export type ActionsStatus = 'unknown' | 'waiting' | 'running' | 'success' | 'failure' | 'cancelled' | 'skipped' | 'blocked';
+
+export type ActionsStepLogLine = {
```

Comment:
I am sure it is only privately used, is it really worth to move everything into a global "types" package?

Reply:
This one likely is only for the actions log yes. The `ActionsStatus`, I could see being used elsewhere too.